jobs/JRRFS_MPASSIT

```diff
@@ -26,10 +26,13 @@ export HISTORY_INTERVAL=${HISTORY_INTERVAL:-1}
 #-----------------------------------------------------------------------
 #
 export pid=${pid:-$$}
+export RUN="rrfs"${WGF}
 if [[ -z "${ENS_INDEX}" ]]; then
-  export RUN="rrfs"
+  export MEMID=""
+  export MEMDIR=""
 else
-  export RUN="ensrrfs"
+  export MEMID="_m${ENS_INDEX}"
+  export MEMDIR="/m${ENS_INDEX}"
 fi
 export jobid=${jobid:-${RUN}_mpassit_g${GROUP_INDEX}_${cyc}}
```

Comment:
can you please change this to have "MEMID" information, similar to other scripts?
export jobid=${jobid:-${RUN}_mpassit${MEMID}g${GROUP_INDEX}${cyc}}

Reply:
Done